src/hooks/useContributions.ts

```diff
@@ -0,0 +1,58 @@
+import { useMemo } from 'react';
+
+import { useQueries } from 'react-query';
+
+import { useSelectedCircle } from 'recoilState';
+
+import { useCurrentCircleIntegrations } from './gql';
+
+interface Contribution {
+  title: string;
+  link: string;
+}
+
+export interface ContributionUser {
+  address: string;
+  contributions: Contribution[];
+  contribution_details_link: string;
+}
+
+interface Response {
+  users: ContributionUser[];
+}
+
+export function useContributionUsers(): ContributionUser[] {
+  const { integrations } = useCurrentCircleIntegrations();
+  const epoch = useSelectedCircle().circleEpochsStatus.currentEpoch;
+
+  const responses = useQueries(
+    epoch
+      ? integrations
+          .filter(i => i.type === 'dework')
+          .map(i => ({
+            queryKey: `circle-integration-contributions-${i.id}-${epoch.id}`,
+            queryFn: () =>
+              fetch(
+                `https://api.dework.xyz/integrations/coordinape/${i.data.organizationId}?epoch_start=${epoch.start_date}&epoch_end=${epoch.end_date}`
```

Comment:
if this fetch errors, will it break the page?

Reply:
No, it will simply _not_ show the Dework tasks and fall back to showing the epoch statement if it exists